x/crosschain/keeper/grpc_query_cctx_rate_limit.go

```diff
@@ -114,23 +121,29 @@ LoopBackwards:
 			endNonce = 0
```

Comment:
This case not covered by tests, I think it might be great to cover it 

https://app.codecov.io/gh/zeta-chain/node/pull/2060/blob/x/crosschain/keeper/grpc_query_cctx_rate_limit.go#L121

Reply:
https://github.com/zeta-chain/node/pull/2060/commits/f9eed1570eccc447230fe15c25ec99d41f1946fb